js/menu.js

```diff
@@ -0,0 +1,32 @@
+function showMenu() {
+  const mobileMenu = document.querySelector('#mobile-menu');
+  const img = document.querySelector('#menu-btn img');
+  const a = document.querySelector('#header a');
+
+  mobileMenu.style.display = 'block';
+  img.src = './imgs/x.svg';
+  a.style.opacity = 0;
+}
+
+function hideMenu() {
+  const mobileMenu = document.querySelector('#mobile-menu');
+  const img = document.querySelector('#menu-btn img');
+  const a = document.querySelector('#header a');
+
+  mobileMenu.style.display = 'none';
+  img.src = './imgs/IconMenu.svg';
+  a.style.opacity = 1;
+}
+
+function toggleHamburgerBtn() { // eslint-disable-line no-unused-vars
+  const mobileMenu = document.querySelector('#mobile-menu');
+<<<<<<< HEAD
+=======
+  
+>>>>>>> 983b7bd52daae6d01d31ab223704f94463d6184f
```

Comment:
- [ ] The syntax you have on line 23 to 26 are not javascript syntax, hence, it is preventing your code from working as expected. After I remove them, your menu Icon click event `showMenu` and `hideMenu` started working properly. Please fix this.

Outside the above issue, you implemented the menu feature professionally, particularly that your functions are easy to follow. Well done 👏🏾